### PR TITLE
Explaining philosophy and process in README.md

### PR DESCRIPTION
**Pull request by gvwilson**
Beefing up the README with the next instructor(s) and contributor(s) in mind.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,203 @@
...
++++
+<!-- .slide: data-background="../content/images/answer-slide.jpg" -->
+
+## Answer
+~~~
+
+The first line after `+++` includes the "answer" background image.
+The sub-slide's title should always be "Answer".
+
+## For Each Training Class
```
Comment (jamesbs) [inline README.md]: May need to revisit this section once we get Github pages setup

File: README.md [hunk 2]
```diff
@@ -1,24 +1,203 @@
...
 
-To see the slides in your browser, go to [http://localhost:8081](http://localhost:8081/#/)
+1.  Modify `./slides/src/content/introduction.md` to include the biographies of the instructors,
+    the class's schedule, and other specific information.
+    Please do *not* include these changes in any PRs you submit to improve the core material.
 
-To modify or add more diagrams go to [this](https://drive.google.com/drive/u/0/folders/0B6Cn_hQVgj2PRHZHQTFxd21UX00) Google Drive folder.
+1.  After starting the server, generate a PDF of your slides by going to
+    [http://localhost:8081/?print-pdf](http://localhost:8081/?print-pdf)
+    and using Google Chrome's print utility (Save as PDF).
```
Comment (jamesbs) [inline README.md]: I might have missed it but was there a request from marketing regarding a PDF version or are we driving this? I haven't seen any demand for PDF versions of the slide content

File: README.md [hunk 3]
```diff
@@ -1,24 +1,203 @@
...
+
+If you edit the slides' source,
+the browser automatically redisplays within a second or two.
+
+## How to Edit
+
+1.  The overall directory structure of `./slides/` is fixed by [Reveal.js][reveal].
+    Please do not make wholesale changes without careful testing.
+
+1.  Slide content is stored in multiple Markdown files in `./slides/src/content`.
```
Comment (jamesbs) [inline README.md]: Might want to mention that markdown files separate slide content per section.

File: README.md [hunk 4]
```diff
@@ -1,24 +1,203 @@
...
+
+1.  Go into the project's `slides` directory.
+
+1.  Run `npm install` to install the JavaScript packages needed by [Reveal.js][reveal].
+
+You only need to do these steps once.
+After that,
+you can view the slides at any time:
+
+1.  Run `npm start` in the `./slides/` directory to run a local server.
```
Comment (jamesbs) [inline README.md]: May want to mention that server is set to run on port 8081 and will error if that port is in use

File: README.md [hunk 5]
```diff
@@ -1,24 +1,203 @@
-# Angular Training Slides
+# Angular Training Materials
 
-Slides for the Angular 2+ training created with Reveal.js and using Markdown format
+This repository contains slides and other materials
```
Comment (jamesbs) [inline README.md]: Are we doing manual line breaks for content?  I think we should probably be consistent on this

File: README.md [hunk 6]
```diff
@@ -1,24 +1,203 @@
...
+1.  It slows the instructor down: if she has to type in the program as
+    she goes along, she can only go twice as fast as her learners,
+    rather than ten times faster as she could with slides.
+
+1.  Learners get to see instructors' mistakes *and how to diagnose
+    and correct them*. Novices are going to spend most of their time
+    doing this, but it's left out of most textbooks.
+
+1.  Watching instructors make mistakes shows learners that it's all
+    right to make mistakes of their.  Most people model the behavior of
```
Comment (codebyuma) [inline README.md]: typo, missing the word "own" - should be "make mistakes of their own"

File: README.md (hunk 5 above)
Comment (gvwilson) [inline README.md]: No, I just prefer semantic linebreaks (leads to fewer errors when moving chunks of text around). Markdown renders all consecutive lines w/out separating blanks as one para.

File: README.md (hunk 4 above)
Comment (gvwilson) [inline README.md]: addressed

File: README.md (hunk 3 above)
Comment (gvwilson) [inline README.md]: addressed

File: README.md (hunk 1 above)
Comment (gvwilson) [inline README.md]: Yup.

File: README.md (hunk 2 above)
Comment (gvwilson) [inline README.md]: For handouts - I'll ad a note saying "if asked to".

File: README.md (hunk 6 above)
Comment (gvwilson) [inline README.md]: fixed.

**Pull request merged**